semantic_conventions/trace/rpc.yaml

```
@@ -57,6 +57,16 @@ groups:
           - net.peer.ip
           - net.peer.name
       - include: network
+
+  - id: rpc.server
+    prefix: rpc
+    extends: rpc
+    span_kind: server
+    brief: 'Semantic Convention for RPC Server'
```

```suggestion
    brief: 'Semantic Convention for RPC Servers'
```
(or is the plural uncommon here?)

---

updated, I will send a follow-up PR to sync similar wording in `http.yaml`